restore.go

```diff
@@ -143,6 +144,20 @@ func restoreContainer(context *cli.Context, spec *specs.LinuxSpec, config *confi
 	return handler.forward(process)
 }
 
+func handleContainerstatus(status libcontainer.Status, container libcontainer.Container, context *cli.Context) (int, error) {
```

Comment:
this is should have multiple return results, because you are not using the first one.